src/chrome/chromeDebugAdapter.ts

```diff
@@ -643,7 +643,7 @@ export class ChromeDebugLogic {
     }
 
     protected threadName(): string {
-        return 'Thread ' + ChromeDebugLogic.THREAD_ID;
+        return localize('thread.prefix', 'Thread ') + ChromeDebugLogic.THREAD_ID;
```

Comment:
Space should go outside of the localized string so it isn't lost in localization

Reply:
Actually the id should be an arugument like {0} in case it goes before "Thread" in some language

Reply:
Changed to return localize('thread.prefix', 'Thread {0}', ChromeDebugLogic.THREAD_ID);